### PR TITLE
Add scaffold files for initial page. Hello, world :D

### PR DESCRIPTION
**Pull request by cchanningallen**
* Adds scaffold files for initial page. Just a starting point; we'll build from here.
* Also, I love you 🤗

### CODE REVIEW

File: src/carter-mc-barter-narter-is-dope.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+body {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+div {
+
```
Comment (b-carter-allen) [inline src/carter-mc-barter-narter-is-dope.css]: Why is there nothing here? This isn't Nam there are rules.

File: src/carter-mc-barter-narter-is-dope.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    <div>
+      Hello, Carter & Channing's world!
+    </div>
+
+    <h1>Title</h1>
+
+    <h3>Subtitle</h3>
+
+    <div>
+
```
Comment (b-carter-allen) [inline src/carter-mc-barter-narter-is-dope.html]: Empty? Rules.

File: src/carter-mc-barter-narter-is-dope.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <h1>Title</h1>
+
+    <h3>Subtitle</h3>
+
+    <div>
+      
+    </div>
+
+    <div>
+
```
Comment (b-carter-allen) [inline src/carter-mc-barter-narter-is-dope.html]: Still empty.

**Review by b-carter-allen — CHANGES_REQUESTED** [review]

File: src/carter-mc-barter-narter-is-dope.css (hunk 1 above)
Comment (cchanningallen) [reply src/carter-mc-barter-narter-is-dope.css]: Ah shiiii gud point

Comment (cchanningallen) [reply src/carter-mc-barter-narter-is-dope.css]: 1 sec, will RM 🔪

File: src/carter-mc-barter-narter-is-dope.html (hunk 2 above)
Comment (cchanningallen) [reply src/carter-mc-barter-narter-is-dope.html]: dood u lrdy sed this 🙄

File: src/carter-mc-barter-narter-is-dope.html (hunk 3 above)
Comment (cchanningallen) [reply src/carter-mc-barter-narter-is-dope.html]: omfg I KEEL EW

**Comment by cchanningallen** [comment]
@b-carter-allen thanks fo the thoughtful feedback! Changes implemented - we G2G?

**Review by b-carter-allen — APPROVED** [review]
LGTM 🦄

**Review by b-carter-allen — APPROVED** [review]

**Review by b-carter-allen — APPROVED** [review]

**Review by b-carter-allen — APPROVED** [review]

**Pull request merged**